### DISCUSSION TITLE
how to add wallpaper to default list

### DISCUSSION THREAD

**Discussion by IPdotSetAF** [rocksdanister/lively#2231] [Q&A]
hey.

i developed a web based wallpaper [https://github.com/IPdotSetAF/NeoMatrix](https://github.com/IPdotSetAF/NeoMatrix) and i was curious if i can add it to the main page of lively (if i actually made it to work with lively that is)

**Comment by rocksdanister** [comment]
Looks really good, the most advanced version of Matrix wallpaper I have seen so far 👍

If you add support for Lively I will try to add it in future software update.

**Reply by wolfwpk** [reply]
I am also developing a wallpaper that integrates live webcams of some ski resorts. I am seeking a way to compile it into the installer as one of the default wallpapers, so that my friends don't have to import extra files when i share it. I am not able to find any default lists in the source code, and the Gallery is not released yet. Therefore, is there a way to customize the default list and installer, which is also a good way to spread the word. Thanks!

**Reply by rocksdanister** [reply]
The default list is managed by me.

Webcam should not work since the permission is disabled by default I think.
